testing.ts

```diff
@@ -0,0 +1,43 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Are you using this file anywhere? Why did you commit it?

Reply:
**[DONE]** Haha! Yes, it was not intended. Removed it now